### PR TITLE
Silencing KubeAPIErrorBudgetBurn alert

### PR DESCRIPTION
**Pull request by anispate**
Silencing KubeAPIErrorBudgetBurn alert as part of  [OSD-20058](https://issues.redhat.com//browse/OSD-20058)

### CODE REVIEW

File: controllers/secret_controller.go [hunk 1]
```diff
@@ -447,8 +447,8 @@ func createSubroutes(namespaceList []string, receiver receiverType) *alertmanage
 		// https://issues.redhat.com/browse/OSD-3326
 		{Receiver: receiverCommon, Match: map[string]string{"cluster": "elasticsearch", "prometheus": "openshift-monitoring/k8s"}},
 		// Route KubeAPIErrorBudgetBurn to PD despite lack of namespace label
```
Comment (fahlmant) [inline controllers/secret_controller.go:449]: This comment needs updating if the intent it to not get this alert.

Comment (clcollins) [inline controllers/secret_controller.go:449]: Remove outdated comments:

```
		// Route KubeAPIErrorBudgetBurn to PD despite lack of namespace label
		// https://issues.redhat.com/browse/OSD-8006 && 
```

Comment (clcollins) [reply controllers/secret_controller.go:449]: Otherwise, LGTM

**Review by clcollins — CHANGES_REQUESTED** [review]

**Review by clcollins — APPROVED** [review]
/lgtm

**Pull request merged**